packages/augur-state/src/db/DB.ts

```diff
@@ -104,29 +102,56 @@ export class DB<TBigNumber> {
    * @param {number} blockstreamDelay Number of blocks by which blockstream is behind the blockchain
    */
   public async sync(augur: Augur<TBigNumber>, chunkSize: number, blockstreamDelay: number): Promise<void> {
-    // Sync generic event types & user-specific event types
+    // Sync generic event types
     let dbSyncPromises = [];
     for (let dbIndex in this.syncableDatabases) {
-      dbSyncPromises.push(this.syncableDatabases[dbIndex].sync(augur, chunkSize, blockstreamDelay, this.defaultStartSyncBlockNumber));
-      // TODO Set indexes
+      dbSyncPromises.push(this.syncableDatabases[dbIndex].sync(augur, chunkSize, blockstreamDelay, this.syncStatus.defaultStartSyncBlockNumber));
     }
+    // TODO Figure out a way to handle concurrent request limit of 40
```

Comment:
Maybe a pool object?

Reply:
I'd like to find out what this is about. It is it just something in the test framework, or was this perhaps something like hitting the max listeners on an EventEmitter?

Reply:
It's possible it may have been hitting the maximum listeners. I'll spend some time today trying to reproduce this and get more info.

Reply:
Created https://github.com/AugurProject/augur/issues/1064 to track this so we can revisit it later.